payment-service/src/main/java/com/hedvig/paymentservice/serviceIntergration/memberService/MemberService.java

```diff
@@ -2,8 +2,12 @@
 
 import com.hedvig.paymentservice.serviceIntergration.memberService.dto.Member;
 import com.hedvig.paymentservice.serviceIntergration.memberService.dto.SanctionStatus;
+import com.hedvig.paymentservice.web.dtos.PaymentProvider;
+
+import java.util.List;
 import java.util.Optional;
 
 public interface MemberService {
   Optional<Member> getMember(String memberId);
+  List<String> getMembersConnectedToProvider(PaymentProvider paymentProvider);
```

Comment:
```suggestion
  List<String> getMembersByPayinProvider(PayinProvider paymentProvider);
```

Lets use the term Payin - Provider in payment-service. There is already an enum for those.